Integrations/Gmail/Gmail.py

```diff
@@ -1028,22 +1333,361 @@ def remove_filter(user_id, _id):
+        transientFileCID = None
+
+    attachments = collect_attachments(entry_ids, file_names)
+    manual_attachments = collect_manual_attachments()
+    transientAttachments = transient_attachments(transientFile, transientFileContent, transientFileCID)
+
+    attachments = attachments + htmlAttachments + transientAttachments + inlineAttachments + manual_attachments
+    attachment_handler(message, attachments)
+
+    encoded_message = base64.urlsafe_b64encode(message.as_string())
+    command_args = {
+        'userId': emailfrom,
+        'body': {
+            'raw': encoded_message,
+        }
+    }
+    service = get_service('gmail', 'v1', additional_scopes=['https://www.googleapis.com/auth/gmail.compose',
+                                                            'https://www.googleapis.com/auth/gmail.send'])
+    result = service.users().messages().send(**command_args).execute()
+    return result
+
+
+def send_mail_command():
+    args = demisto.args()
+    emailto = argToList(args.get('to'))
+    emailfrom = args.get('from')
+    body = args.get('body')
+    subject = args.get('subject')
+    entry_ids = argToList(args.get('attachIDs'))
+    cc = argToList(args.get('cc'))
+    bcc = argToList(args.get('bcc'))
+    htmlBody = args.get('htmlBody')
+    replyTo = args.get('replyTo')
+    file_names = argToList(args.get('attachNames'))
+    attchCID = argToList(args.get('attachCIDs'))
+    transientFile = argToList(args.get('transientFile'))
+    transientFileContent = argToList(args.get('transientFileContent'))
+    transientFileCID = argToList(args.get('transientFileCID'))
+    additional_headers = argToList(args.get('additionalHeader'))
+    template_param = args.get('templateParams')
+
+    if emailfrom is None:
+        emailfrom = ADMIN_EMAIL
+
+    result = send_mail(emailto, emailfrom, subject, body, entry_ids, cc, bcc, htmlBody,
+                       replyTo, file_names, attchCID, transientFile, transientFileContent,
+                       transientFileCID, additional_headers, template_param)
+    return sent_mail_to_entry('Email sent:', [result], emailto, emailfrom, cc, bcc, htmlBody, body, subject)
+
+
+'''FETCH INCIDENTS'''
+
+
 def fetch_incidents():
     params = demisto.params()
     user_key = params.get('queryUserKey')
     user_key = user_key if user_key else ADMIN_EMAIL
     query = '' if params['query'] is None else params['query']
     last_run = demisto.getLastRun()
-    last_fetch = last_run.get('time')
-
-    # handle first time fetch
+    last_fetch = last_run.get('gmt_time')
+    # handle first time fetch - gets current GMT time -1 day
     if last_fetch is None:
-        last_fetch = datetime.now() - timedelta(days=1)
-    else:
-        last_fetch = datetime.strptime(
-            last_fetch, '%Y-%m-%dT%H:%M:%SZ')
-    current_fetch = last_fetch
+        last_fetch = (datetime.utcfromtimestamp(int(time.time())) - timedelta(days=FETCH_TIME)).isoformat() + 'Z'
```

Comment:
be consistent with other integrations, use `parse_date_range` on the value of `FETCH_TIME`